chopper/test/request_test.dart

```diff
@@ -92,53 +92,55 @@ void main() {
   group('Request.uri', () {
     test('constructor produces a BaseRequest', () {
       expect(
-        Request.uri('GET', Uri.parse('https://foo/bar')),
+        Request.uri('GET', Uri.parse('https://foo/bar'), ''),
         isA<http.BaseRequest>(),
       );
     });
 
     test('method gets preserved in BaseRequest', () {
       expect(
-        Request.uri('GET', Uri.parse('https://foo/bar')).method,
+        Request.uri('GET', Uri.parse('https://foo/bar'), '').method,
         equals('GET'),
       );
     });
 
     test('url is correctly parsed and set in BaseRequest', () {
       expect(
-        Request.uri('GET', Uri.parse('https://foo/bar')).url,
+        Request.uri('GET', Uri.parse('https://foo/bar'), '').url,
         equals(Uri.parse('https://foo/bar')),
       );
 
       expect(
-        Request.uri('GET', Uri.parse('https://foo/bar?lorem=ipsum&dolor=123'))
+        Request.uri(
+                'GET', Uri.parse('https://foo/bar?lorem=ipsum&dolor=123'), '')
             .url,
         equals(Uri.parse('https://foo/bar?lorem=ipsum&dolor=123')),
       );
 
       expect(
         Request.uri(
           'GET',
-          Uri.parse('https://foo/bar'),
-          parameters: {
+          Uri(scheme: 'https', host: 'foo', path: 'bar', queryParameters: {
             'lorem': 'ipsum',
-            'dolor': 123,
-          },
+            'dolor': '123',
+          }),
+          '',
         ).url,
         equals(Uri.parse('https://foo/bar?lorem=ipsum&dolor=123')),
       );
 
       expect(
         Request.uri(
           'GET',
-          Uri.parse(
-            'https://foo/bar?first=sit&second=amet&first_list=a&first_list=b',
```

Comment:
This test here for example merges a Uri with existing parameters with an additional Map from the `Request.uri` constructor's `parameters` parameter.